README.md

```diff
@@ -10,9 +10,9 @@ The SuperTuxKart homepage can be found at <https://supertuxkart.net/>. There is
 ## Hardware Requirements
 To run SuperTuxKart, make sure that your computer's specifications are equal or higher than the following specifications:
 
-* A graphics card capable of 3D rendering - NVIDIA GeForce 8 series and newer (GeForce 8100 or newer), AMD/ATI Radeon HD 4000 series and newer, Intel HD Graphics 3000 and newer. OpenGL >= 3.3
+* A graphics card capable of 3D rendering - NVIDIA GeForce 400 series and newer (GeForce GT 430 or newer), AMD/ATI Radeon HD 5000 series and newer, Intel HD Graphics 3000 and newer. OpenGL >= 3.3
 * You should have a CPU that's running at 1 GHz or faster. 
-* You'll need at least 512 MB of free VRAM (video memory).
+* Minimum system memory: 2 GB
```

Comment:
You changed a VRAM minimal requirement into a RAM requirement, why? Both are relevant, but they are not the same thing. (or maybe move the VRAM part with the GPU requirements section)